fitnesse/resources/bootstrap-plus/js/bootstrap-plus.js

```diff
@@ -585,4 +592,54 @@ function deleteTag(successData, neededValues) {
 }
 /*
     DELETE END | ADD & DELETE TAGS FUNCTIONS END
-*/
\ No newline at end of file
+*/
+
+function getVersionData(callback, url) {
+     // const $ = require('jquery');
+    $.ajax({
+        type: 'GET',
+        url: url,
+        contentType: 'charset=utf-8',
+        success: data => callback(data),
+        error: function (xhr) {
+            console.log('Error code for version checker: ' + xhr.status, xhr);
+        }
+    });
+}
+
+function versionCheck(data) {
+    if (data != undefined) {
+        data.forEach(versionData => {
+            // Replace property 'version' with 'currentVersion' to make al the property names alike
+            if (versionData.hasOwnProperty('version')) {
+                versionData["currentVersion"] = versionData['version'];
+                delete versionData['version'];
+            }
+
+            // Make version status
+            // Clean string so it can be used as an int for version comparison
+            versionData['formatCurrentVersion'] = parseInt(versionData.currentVersion.replace(/\D/g, ''));
+            versionData['formatLatestVersion'] = parseInt(versionData.latest.replace(/\D/g, ''));
+
+            if (versionData.formatCurrentVersion <= versionData.formatLatestVersion) {
+                if (versionData.formatCurrentVersion < versionData.formatLatestVersion) {
```

Comment:
dit gaat niet altijd goed denk ik.. 1.3.0 is in deze vergelijking achter op 1.2.2.0 (130 < 1220). Zal met name bugfixversies raken. Toolchain-plugin had bijvoorbeeld 1.18 -> 1.18.1 -> 1.19 als versievolgorde.

Ik denk dat hier iets meer logica nodig is. (split op punt in major, minor, patch bijvoorbeeld en dan alleen kijken naar patch als major en minor gelijk zijn, en alleen kijken naar minor als major gelijk is, etc. Zou dat in een recursive methode proppen om de eersvolgende wijsneus met versie a.b.c.d te slim af te zijn.